edx_solutions_projects/views.py

```diff
@@ -83,6 +83,23 @@ def create(self, request):
 
         return response
 
+    def destroy(self, request, pk):
+        """
+        Delete a workgroup and its cohort.
+        """
+        work_group = self.get_object()
+        course_key = get_course_key(work_group.project.course_id)
+        cohort = get_cohort_by_name(course_key, work_group.cohort_name)
+        response = super(WorkgroupsViewSet, self).destroy(request, pk)
+        if response.status_code == status.HTTP_204_NO_CONTENT:
+            WorkgroupUser.objects.filter(workgroup_id=pk).delete()
```

Comment:
Why do we need to delete this explicitly? By foreign key relation I assume it will be cascade deleted when `Workgroup` is deleted?